webrender/src/box_shadow.rs

```diff
@@ -194,7 +195,7 @@ impl FrameBuilder {
                         // case, we ensure the size of each corner is the same,
                         // to simplify the shader logic that stretches the blurred
                         // result across the primitive.
-                        radii_kind = BorderRadiusKind::NonUniform;
+                        image_kind = BrushImageKind::NinePatch;
                         let max_width = shadow_radius.top_left.width
```

Comment:
not directly related to the PR, but we could really use some [TypedPoint2D::max](https://docs.rs/euclid/0.16.0/euclid/struct.TypedPoint2D.html#method.max) here

Reply:
This is a ```TypedSize``` which doesn't appear to have a ```max``` implementation (we should fix that though in euclid).